5.5/mysql/bin/entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+function usage {
+	echo "You must specify following environment variables:"
+	echo "  \$MYSQL_ROOT_PASSWORD"
+	echo "  \$MYSQL_USER"
+	echo "  \$MYSQL_PASSWORD"
+	echo "  \$MYSQL_DATABASE"
+	exit 1
+}
+
+test -z "$MYSQL_ROOT_PASSWORD" && usage
+test -z "$MYSQL_USER" && usage
+test -z "$MYSQL_PASSWORD" && usage
+test -z "$MYSQL_DATABASE" && usage
+
+export X_SCLS="mysql55"
+source /opt/rh/mysql55/enable
+
+if [ ! -d '/var/lib/mysql/data' -a "$1" = 'mysqld_safe' ]; then
+
+	echo 'Running mysql_install_db ...'
+	mysql_install_db --user=mysql --datadir=/var/lib/mysql/data
+	echo 'Finished mysql_install_db'
```

Comment:
I don't see the value of having 2 messages here other than you want to do a benchmark, in that case it is missing timestamps ;-)

Easiest would be to just print the one on line 22 and nuke the second